### PR TITLE
Add exit action

### LINKED ISSUE #6: Exit the interactive menu
To exit the menu, I currently need to hit ctrl c. It would be nice to have a way to exit via the CLI.

### PR DESCRIPTION
**Pull request by scottrigby**
Fixes #6 

This adds a working `[Exit]               Exit` to the end of each prompt.

### CODE REVIEW

File: explorer/explorer_cluster.go [hunk 1]
```diff
@@ -47,6 +49,9 @@ func (n *ClusterExplorer) Execute(selection string) error {
 			PreviousExplorer:   n,
 		}
 		return Explore(namespaceExplorer)
+	case exitLabel:
+		os.Exit(0)
```
Comment (kris-nova) [inline explorer/explorer_cluster.go]: Can we pull the exit out into a function so we can call it from other areas of the code? Im working on a signal handler that would want to call said function

Comment (scottrigby) [reply explorer/explorer_cluster.go]: niice. Ok will do

**Comment by scottrigby** [comment]
PS, I played with https://github.com/thecodeteam/goodbye locally (just to add a “kthxbye!” message. It works, but that project seems pretty unnecessary here since we only reach it with a prompt, and chzyer/readline already adds `✔ [Exit]               Exit` 🤷‍♂️

**Comment by kris-nova** [comment]
w00t!

**Pull request merged**